x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list_notify_badge.tsx

```diff
@@ -79,6 +84,26 @@ const getNextRuleSnoozeSchedule = (rule: { snoozeSchedule?: RuleSnooze }) => {
   return nextSchedule;
 };
 
+// Gets the next occurrence date string relative to the current date
+// Falls back to dtstart in case there is no more next occurrence
+const getNextOccurrenceDateString = (rRule: SnoozeRRule) => {
+  if (moment().isBefore(rRule.dtstart)) {
+    return rRule.dtstart;
+  }
+  try {
+    const recurrenceRule = getRRuleFromSnooze(rRule);
```

Comment:
Is there any way to do this computation server-side so that we don't have to expose `rrule` to the frontend and balloon the bundle size?